webRepo.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+#if [ $TRAVIS_BRANCH == 'master' ]; then
+  # push new versions of compiled libraries
+  cd /
```

Comment:
do we need to work in root directory ?
Why not using any of convenience directories ?
https://docs.travis-ci.com/user/environment-variables/#convenience-variables i.e. `TRAVIS_BUILD_DIR` ?

Reply:
Because in this case TRAVIS_BUILD_DIR is where DockerFile repo is, and we want clone another one (web repo) I thought is is bad idea to keep one repo inside another

Reply:
OK, I see the point now - cloning inside another repo may lead to some troubles. Can you add some comments mentioning that this was the reason to use another directory ?

But still do you see some way to avoid using / ? What about `$TEMP` ?

Reply:
Yes i agree it not the base idea use /, i will use $TEMP